src/common/store/pages/audio-rewards/slice.ts

```diff
@@ -112,6 +113,27 @@ const slice = createSlice({
         current_step_count: stepCount
       }
     },
+    resetUserChallengeCurrentStepCount: (
+      state,
+      action: PayloadAction<{
+        challengeId: ChallengeRewardID
+      }>
+    ) => {
+      const { challengeId } = action.payload
+      const userChallengeOverride =
+        state.userChallengesOverrides[challengeId] || {}
+      if ('current_step_count' in userChallengeOverride) {
+        if (Object.keys(userChallengeOverride).length > 1) {
+          // eslint-disable-next-line
```

Comment:
Why is this disabled?

Reply:
it's because the variable `current_step_count` is not used causing a lint error

Reply:
ahh gotcha, can we disable just that rule?

Reply:
just set to undefined